Workbooks/Virtual Machines/Virtual Machine Performance/Single Virtual Machine Performance.workbook

```diff
@@ -0,0 +1,515 @@
+{
+  "version": "Notebook/1.0",
+  "isLocked": false,
```

Comment:
for `isLocked` right here. you want it to be true, otherwise when the new workbook is created, it will be in "edit" mode instead of the normal "viewer" mode.